### ISSUE TITLE
Fabric deployment tab

### ISSUE THREAD

**Issue by gashcrumb** [hawtio/hawtio#1059]
As Claus pointed out it seems like the profile UI isn't the best for a first-time user of fabric, it requires a level of familiarity with how fabric works that folks new to fabric don't have.  Typically folks are really going to just want to deploy their app to a fabric somehow, and the first step really probably shouldn't be manually configuring a profile.  So maybe it'd make sense to kick off a deployment tab, where a user can upload an artifact (or multiple artifacts), we can then scan the artifact for dependencies (pull in a bit of FAB I think) and suggest what features or bundles would be needed and then generate a profile from this wizard and upload the user's artifacts to the fabric via the fabric maven proxy.

So basically I'm thinking we've a file upload widget already that can be used to move jars or whatnot from the user's desktop to a temporary staging area.  As artifacts are uploaded we can scan them using FAB and start generating the start of a profile, i.e. we can suggest what else would be needed to deploy a given artifact.

**Comment by davsclaus** [comment]
closing fabric v1 tickets, ENH requires is tracked on jboss fuse side.
